.github/BOTMETA.yml

```diff
@@ -619,207 +684,306 @@ files:
+  lib/ansible/plugins/lookup/__init__.py:
+    support: core
   lib/ansible/plugins/lookup/conjur_variable.py:
     maintainers: $team_cyberark_conjur
     notified: cyberark-bizdev
+  lib/ansible/plugins/lookup/dig:
+    maintainers: jpmens
+    labels: community
   lib/ansible/plugins/lookup/cyberarkpassword.py:
     notified: cyberark-bizdev
-  lib/ansible/plugins/netconf/:
-    maintainers: $team_networking
-    labels: networking
+  lib/ansible/pluginis/lookup/nios:
+    support: core
+    maintainers: $team_networking sganesh-infoblox
+    labels:
+      - networking
+      - infoblox
   lib/ansible/plugins/lookup/onepassword.py:
     maintainers: samdoran
     ignored: azenk
   lib/ansible/plugins/lookup/onepassword_raw.py:
     maintainers: samdoran
     ignored: azenk
+###############################
+# plugins/netconf
+  lib/ansible/plugins/netconf/:
+    support: network
+    maintainers: $team_networking
+    labels: networking
   lib/ansible/plugins/netconf/sros.py:
     maintainers: wisotzky $team_networking
     labels: networking
+###############################
+# plugins/shell
+  lib/ansible/plugins/shell/:
+    support: core
   lib/ansible/plugins/shell/powershell.py:
+    support: core
     maintainers: $team_windows_core
     labels:
-    - windows
-  lib/ansible/plugins/terminal/aireos.py:
-    maintainers: jmighion
-    labels: networking
-  lib/ansible/plugins/terminal/aruba.py:
-    maintainers: jmighion
-    labels: networking
+      - windows
+###############################
+# plugins/strategy
+  lib/ansible/plugins/strategy/:
+    support: core
+###############################
+# plugins/terminal
+  lib/ansible/plugins/terminal/:
+    labels: networking
+  lib/ansible/plugins/terminal/__init__.py:
+    support: network
   lib/ansible/plugins/terminal/asa.py:
     maintainers: ogenstad gdpak
-    labels: networking
   lib/ansible/plugins/terminal/dellos10:
     maintainers: skg-net
-    labels: networking
   lib/ansible/plugins/terminal/edgeos.py:
     maintainers: samdoran
-    labels: networking
   lib/ansible/plugins/terminal/eos.py:
+    support: network
     maintainers: $team_networking
-    labels: networking
   lib/ansible/plugins/terminal/exos.py:
     maintainers: rdvencioneck
-    labels: networking
   lib/ansible/plugins/terminal/ios.py:
+    support: network
     maintainers: $team_networking
-    labels: networking
   lib/ansible/plugins/terminal/iosxr.py:
+    support: network
     maintainers: $team_networking
-    labels: networking
   lib/ansible/plugins/terminal/ironware.py:
     maintainers: paulquack
-    labels: networking
   lib/ansible/plugins/terminal/junos.py:
+    support: network
     maintainers: $team_networking
-    labels: networking
   lib/ansible/plugins/terminal/nos.py:
     maintainers: $team_extreme
     labels: networking
   lib/ansible/plugins/terminal/nxos.py:
+    support: network
     maintainers: $team_networking
-    labels:
-    - networking
+    labels: nxos
```

Comment:
No need to add `nxos` since it's a path component. When the path is `lib/ansible/plugins/terminal/nxos.py`, the bot will automatically adds the following labels (if they exists):  `lib`, `ansible`, `plugins`, `terminal`, `nxos` (`.py` and `.ps1` extensions are [removed](https://github.com/ansible/ansibullbot/blob/149f8e099a8d5822805418c62c73f3fd6256371f/ansibullbot/parsers/botmetadata.py#L79-L80).